StocksDashboard.py

```diff
@@ -46,9 +45,21 @@ class Base_StocksDashboard():
     }
     mode = 'vline'
 
-    def __init__(self, **kwargs):
-        for k, v in kwargs.items():
-            setattr(self, k, v)
+    width = WIDTH
+    height = HEIGHT
+    ncols = 1
+    names = None
+
+    # code = "source.set('selected', cb_data['index']);"
+    # callback = CustomJS(args={'source': source}, code=code)
+
+    def __init__(self, width=None, height=None, ncols=None):
```

Comment:
You could do: 
```
def __init__(self, width=WIDTH, height=HEIGHT, ncols=1):
    self.width = width
    self.height = height
    self.ncols = ncols
```
Then you can remove the declaration of the members of the class (line 48...).

Reply:
ok, but then we should check these values somehow.